File: test/fixtures/ruby/put_with_T_option.rb
require 'net/http'

uri = URI('http://localhost:28139/twitter/_mapping/user?pretty')
req = Net::HTTP::Put.new(uri)
req.content_type = 'application/json'

req.body = File.read('my_file.txt')

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end

File: test/fixtures/node/put_with_T_option.js
import fetch, { fileFromSync } from 'node-fetch';

fetch('http://localhost:28139/twitter/_mapping/user?pretty', {
  method: 'PUT',
  headers: {
    'Content-Type': 'application/json'
  },
  body: fileFromSync('my_file.txt')
});
